common/test/acceptance/pages/studio/settings.py

```diff
@@ -29,7 +30,8 @@ def entrance_exam_field(self):
         """
         Returns the enable entrance exam checkbox.
         """
-        return self.q(css='#entrance-exam-enabled').execute()
+        self.wait_for_element_presence('#entrance-exam-enabled', 'Waiting for entexam checkbox...')
+        return self.q(css='#entrance-exam-enabled')
```

Comment:
q actually returns a BrowserQuery object, not an element. I think you want to return q.results which returns a list of results (or the first element in the list, depending on your intention).

http://bok-choy.readthedocs.org/en/latest/_modules/bok_choy/page_object.html#PageObject.q